gapis/api/vulkan/api/properties_features_requirements.api

```diff
@@ -42,8 +42,10 @@
 
 @override
 cmd VkResult vkEnumerateInstanceExtensionProperties(
+    @len("null-terminated")
```

Comment:
We already have a special type for this, we might want to at least reference that. 
`@str` or similar would make sens.e

Reply:
Done